src/main/java/org/kitteh/irc/client/library/implementation/EventListener.java

```diff
@@ -977,6 +977,13 @@ public void ctcp(ClientReceiveCommandEvent event) {
                         case "FINGER":
                             reply = "FINGER om nom nom tasty finger";
                             break;
+                        case "DCC":
+                            // Handle targeted DCC chat
+                            // this.fire(new DCCRequestEvent());
+                            // TODO how can the parameters be extracted @mbaxter???
```

Comment:
`event.getParameters().get(2)` is going to be the `CHAT`
3 will be ... uh... chat again? wut
4 will be ip
5 will be ... addr? What is your documentation saying? :P